entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+RELAY_TLS_CA=${RELAY_TLS_CA:-/etc/ssl/certs/ca-certificates.crt}
```

Comment:
Thanks for your contribution,
Can you put this default value as a default value for the ENV item in the Dockerfile instead of here please ?

Reply:
Done! Didn't realize all the related env variables were already there when I wrote this